Update key rotation details in secure-file-transfer-protocol-host-keys.md (#128570)
Azure/azure-content: articles/storage/blobs/secure-file-transfer-protocol-host-keys.md

Verification would involve comparing the fingerprint from the client output with the one stored in the table below. If they match, then type `yes` to continue and the client will then automatically store the new key in the `known_hosts` for the future.

### What is the timeline for key rotation?

> When the expiry time in the below table arrived, Azure Storage began the process of rotating host keys across all storage accounts. Different storage accounts will experience the key rotation at different times. All storage accounts should see their host keys rotated within several months.

Reviewer: ```suggestion
When a host key expires, Azure Storage begins the process of rotating host keys across all storage accounts. Different storage accounts will experience the key rotation at different times. All storage accounts should see their host keys rotated within several months.
```

Reviewer: Can we simplify this a bit? Here's a suggestion.

Author: The was initially intentional, because even though we're saying "expires" and "expiry time", that terminology implies behavior that isn't really happening.  I changed it to match your suggestion (thanks!), but the next time we rotate host keys, I think we should use a different term other than "expiry".

### How long does the rotation take?

> Rotations are gradual and may take multiple weeks. Once host key rotation has begun for a storage account, either the old or new host key may be presented by the Azure service during this time. During the rotation, clients could see the old host key being presented even after the new one is first seen.

Reviewer: ```suggestion
Rotations are gradual and might take multiple weeks. Once host key rotation has begun for a storage account, either the old or new host key may be presented by the Azure service during this time. During the rotation, clients could see the old host key being presented even after the new one is first seen.
```

Author: Thanks, fixed.
